src/datastore/postgres-store.ts

```diff
@@ -1,64 +1,65 @@
 import * as path from 'path';
 import { EventEmitter } from 'events';
 import PgMigrate, { RunnerOption } from 'node-pg-migrate';
-import { Pool, PoolClient, ClientConfig, Client, ClientBase, QueryResult, QueryConfig } from 'pg';
```

Comment:
Can all these formatting changes be reverted? I'd be in favor of adding an eslint rule to enforce alphanumeric ordering of imports, but that should be done in a separate PR where we apply it to the whole codebase. In this PR, it just creates noise and merge conflicts.

Reply:
Sure. Reverted